experimental/FTheoryTools/src/types.jl

```diff
@@ -143,3 +143,80 @@ end
   end
 
 end
+
+
+##############################################################################
+# 3 Struct for Quadrillion F-theory Standard Models when read in from database
+##############################################################################
+
+struct QSMModel
```

Comment:
This should be a subtype of AbstractFTheoryModel, probably. And I suppose you intentionally made it immutable?

Reply:
No. This is just a helper type to facilitate reading from the QSM database. This reading process returns an element of this type. I then refactor it into a proper QSM model with set attributes.

Presumably this could be improved in the future.

Reply:
Understood, then it makes sense as is